tests/test_utils/tasks.py

```diff
@@ -1,13 +1,14 @@
 import os
 
 
-def upload_test_file(client, filename='test.jpg'):
+def upload_test_file(client, filename='test.jpg', original_filename='example.jpg'):
     """
     Uploads a file from data dir to the client.
     :param client: where the file should be uploaded to
-    :param filename: name of the test file
+    :param filename: name used during upload of the test file
+    :param original_filename: name of file on the path
     """
-    path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'data/example.jpg')
+    path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'data/%s' % original_filename)
```

Comment:
ugh, you should switch over to [`pathlib`](https://docs.python.org/3/library/pathlib.html)...
```suggestion
    path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'data', original_filename)
```